pkg/hyper/sandbox.go

```diff
@@ -51,6 +79,28 @@ func (h *Runtime) RunPodSandbox(config *kubeapi.PodSandboxConfig) (string, error
 	return podID, nil
 }
 
+func addNetNsInfos2UserPod(userpod *types.UserPod, info *NetNsInfos) {
```

Comment:
Only bridge plugin is supported in this PR, add a TODO of supporting other plugins